frontend/public/style/_overrides.scss

```diff
@@ -40,6 +40,10 @@ small {
   margin: 0 !important;
 }
 
+.margin-bottom {
```

Comment:
This shouldn't be a generic class. I'd recommend

```css
.co-m-pane__explanation--margin-bottom {
```

and move it next to the `.co-m-pane__explanation` declaration. This is more in line with BEM naming. See http://getbem.com/naming/

I'm still a little uncertain why we need to special case this specific page. Can you add a before and after screenshot with the margin changes?

Reply:
Jenny didn't like the visual hierarchy. She had asked for extra margins under the explanation (helptext) -- this class. She had additionally asked for the space between the "Container" header and form fields to be reduced. I had implemented the latter globally the last round of edits and you'd questioned it/she never responded as to whether she wanted that globally, so I took that change out.

With no margin changes:
![screen shot 2018-06-11 at 3 52 59 pm](https://user-images.githubusercontent.com/7014965/41621913-0e641656-73dc-11e8-80a9-0112949a2db8.png)

With margin changes Jenny wanted:
![screen shot 2018-06-11 at 4 10 08 pm](https://user-images.githubusercontent.com/7014965/41621916-0f076b26-73dc-11e8-8426-b318d68d2684.png)

Reply:
Hi @spadgett , 

After checked the screenshot, I feel I'm also in favor of having larger bottom-margin (30px) to have a better visual hierarchy. But instead of creating a new class in _overrides.scss, I am wondering if we can simply increase the margin-bottom of **co-m-pane__explanation** to **30px**? 

Ingress Details page:
[margin-bottom: 20px]
![ori __co-m-pane__explanation___margin-bottom_20px](https://user-images.githubusercontent.com/5903705/41625439-b388fe26-73cd-11e8-9556-a3ecc868537c.png)

[margin-bottom: 30px]
![new __co-m-pane__explanation___margin-bottom_30px](https://user-images.githubusercontent.com/5903705/41625445-b745c508-73cd-11e8-8f98-8dc906c0dc11.png)

Create RoleBinding page:
[margin-bottom: 20px]
![ori_create_binding __co-m-pane__explanation___margin-bottom_20px](https://user-images.githubusercontent.com/5903705/41625449-bb107b6a-73cd-11e8-818c-c0000682080f.png)

[margin-bottom: 30px]
![new_create_binding __co-m-pane__explanation___margin-bottom_30px](https://user-images.githubusercontent.com/5903705/41625456-bdcdedc4-73cd-11e8-9c10-b1004c8daa7d.png)


Also, I am a little bit worried about the help text already mentioned **... You can enter text or add values from a ConfigMap or secret. ** before users can actually "Add Value from Config Map or Secret" on this UI.

Are we going to have **Add Value from Config Map or Secret** for this release?

Reply:
> Are we going to have Add Value from Config Map or Secret for this release?

We're working on it this sprint, but I agree we should probably just take the text out until it's done.

Reply:
Got it, yeah, I don't want to introduce the extra burden here. As long as we'll have the actual functionality to add value from ConfigMap/Secret, I am fine with either way. Please just go with the less problematic one, and thanks!

Reply:
>But instead of creating a new class in _overrides.scss, I am wondering if we can simply increase the margin-bottom of co-m-pane__explanation to 30px?

Yeah, that's what I had in originally; I'll change it back.

What about the headline padding? Jenny wanted "Container hello-openshift" to have margin-bottom: 20px rather than 30px. Is it okay if that's a global change as well?

The class is .co-section-title.

Reply:
Chatted on Slack with @robszumski and @serenamarie125 , we like the margin-bottom: 20px.
The * .co-section-title* is also used on other section title. I've tried with a few titles and it seems fine. 

Let's do that.

*bottom-margin: 30px*
<img width="713" alt="bottom-margin__30px" src="https://user-images.githubusercontent.com/5903705/41791800-b09f544a-760b-11e8-8f1f-2bf97b79b040.png">
<img width="649" alt="screen shot 2018-06-22 at 10 58 56 am" src="https://user-images.githubusercontent.com/5903705/41791850-d289fe34-760b-11e8-9143-c3f69024d981.png">


*bottom-margin: 20px*
<img width="685" alt="bottom-margin__20px" src="https://user-images.githubusercontent.com/5903705/41791801-b0ca1dba-760b-11e8-9cb7-c5e7788ad3eb.png">
<img width="673" alt="screen shot 2018-06-22 at 10 58 41 am" src="https://user-images.githubusercontent.com/5903705/41791841-cc653e60-760b-11e8-9e1a-820a4d4af315.png">

Reply:
Thanks!